pkg/lib/shared/functions.go

```diff
@@ -202,7 +203,7 @@ func GetTlsConfig(opts Options) (*tls.Config, error) {
 	for name, cert := range opts.Certificates {
 		if strings.HasPrefix(name, "extra_ca_certs/") {
 			if ok := rootCAs.AppendCertsFromPEM(cert); !ok {
-				return nil, errors.New("Failed to append custom certificate: " + name)
+				log.Warningf("Could not load extra ca cert file: %s. Skipping.", cert)
```

Comment:
Isn't `cert` the actual certificate content? Wouldn't be better to show the filename instead?

Reply:
que idiota eu sou, updated